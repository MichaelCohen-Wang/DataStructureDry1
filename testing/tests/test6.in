add_ship 349 11
get_cannons 349
update_pirate_treasure 389 20
update_pirate_treasure 36 29
add_ship 265 17
get_cannons 265
get_cannons 331
remove_pirate 36