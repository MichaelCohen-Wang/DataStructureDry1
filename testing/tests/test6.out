add_ship: SUCCESS
get_cannons: SUCCESS, 11
update_pirate_treasure: FAILURE
update_pirate_treasure: FAILURE
add_ship: SUCCESS
get_cannons: SUCCESS, 17
get_cannons: FAILURE
remove_pirate: FAILURE
